test/test_helper.rb

```diff
@@ -28,7 +26,7 @@
   Capybara.register_driver :poltergeist do |app|
     Capybara::Poltergeist::Driver.new(app, {:js_errors => true, :timeout => 60})
   end
-  Capybara.default_wait_time = 30
+  Capybara.default_max_wait_time = 30
```

Comment:
This is a recent (and unrelated) change that would also need a newer version of capybara specified in the Gemfile.